routes/messages/UI/GenericGenerator.js

```diff
@@ -0,0 +1,42 @@
+const generateButtons = require('routes/messages/UI/ButtonGenerator.js');
+console.log(generateButtons);
+
+/**
+ * Dynamically generates messenger buttons
+ * Params:
+ * - "content" (object) = {
+ *  type:String, 
+ *  template_type:String, 
+ *  elements:Array, 
+ *  buttons:Array
+ * }
+ */
+module.exports = async (content) => {
+  const { type, template_type, elements, buttons } = content;
+
+  return {
+    attachment: {
+      type: type,
+      payload: {
+        template_type: template_type,
```

Comment:
Since we plan on making this for generic templates, I think we can default the type and template_type to "template" and "generic" respectively